webhooks/ssp_webhook.go

```diff
@@ -37,9 +42,17 @@ import (
 	sspv1beta2 "kubevirt.io/ssp-operator/api/v1beta2"
 )
 
+// RBAC Rule needed by webhook
+// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingwebhookconfigurations,verbs=list;patch
+
 var ssplog = logf.Log.WithName("ssp-resource")
 
-func Setup(mgr ctrl.Manager) error {
+func Setup(ctx context.Context, mgr ctrl.Manager) error {
+	err := patchValidatingWebhookConfiguration(ctx, mgr)
```

Comment:
Is this prone to flakiness? Is it possible that the webhook server is deployed and running before the `ValidatingWebhookConfiguration` was created?

Reply:
Possibly. I will change the code to use a controller and watch the configuration.